src/CBCNodeListViewController.h

```diff
@@ -67,6 +75,9 @@ FOUNDATION_EXTERN CBCNode *_Nonnull CBCCreateNavigationTree(void);
 /** The children of this node. */
 @property(nonatomic, strong, nonnull) NSArray<CBCNode *> *children;
 
+/** The example you wish to debug as the initial view controller. */
+@property(nonatomic, strong, nullable) CBCNode *debugLeaf;
```

Comment:
In a follow-up change it might be cleaner if we add an `isDebug` state to each node and then provide a separate method for extracting the debug leaf, assuming you also agree that this is reasonable.

Reply:
That is also a viable solution, I had thought of something similar but then thought about the future where we might have a lot of examples out there, and if we made a separate method for extraction, that would need us to make another pass. The navigation tree is pretty flat overall so I am assuming this would be more of a o(N) than a o(logN) pass even if we had isDebug states throughout the branches to the leaf.